src/main/java/com/gluonhq/substrate/model/Triplet.java

```diff
@@ -37,6 +39,18 @@
     private String vendor;
     private String os;
 
+    public static Triplet fromCurrentOS() {
```

Comment:
move it after the other public `Triplet` constructor?

Reply:
The reason it is first is that it is a static factory function, not a constructor

Reply:
Makes sense, somehow I thought it was placed in the middle of both constructors, my bad.